package.json

```diff
@@ -104,6 +105,7 @@
     "eslint": "^8.7.0",
     "eslint-config-3box": "^0.4.0",
     "eslint-plugin-jest": "^25.7.0",
+    "ganache": "^7.3.2",
```

Comment:
Is `ganache` a newer version of `ganache-core`? If so, maybe it makes sense to drop `ganache-core`.